account-kit/infra/src/chains.ts

```diff
@@ -562,3 +562,27 @@ export const gensynTestnet: Chain = defineChain({
   },
   testnet: true,
 });
+
+export const riseTestnet: Chain = defineChain({
```

Comment:
i think this needs to be exported in `account-kit/infra/src/index.ts` as well

like this: https://github.com/alchemyplatform/aa-sdk/commit/8c55523dd435018ded51d3f7af46fa9f6f6bc261